docs/guidance/map/awards_contracts_buyers_suppliers.md

```diff
@@ -16,12 +16,7 @@ The [UNCITRAL Model Law on Public Procurement (2011)](https://uncitral.un.org/en
 
 In OCDS, the `Award` object is intended to communicate a direct relationship between items, suppliers, and values. It ought to be possible to know, at the award stage, in OCDS data, which items will later be supplied by which suppliers, and what the value of those contracts will be.
 
-```{eval-rst}
-.. admonition:: Note
-   :class: note
-
-   .. markdown::
-
+```{note}
       The OCDS schema and documentation are not clear what, precisely, is meant by 'award'. A revision of the definition of `Award` in OCDS is being considered for a future version of the standard ([GitHub issue](https://github.com/open-contracting/standard/issues/895)).
```

Comment:
The indentation of all admonition blocks needs to be fixed. Otherwise, they appear as code blocks, like https://standard.open-contracting.org/staging/eval-rst/en/guidance/map/awards_contracts_buyers_suppliers/